guides/source/caching_with_rails.md

```diff
@@ -512,6 +512,20 @@ class ProductsController < ApplicationController
 end
 ```
 
+Sometimes we want to cache response (static page) that never gets expired.
+To achieve that, we can use `http_cache_forever` helper and by doing so browser
+and proxies should cache it indefinitely.
+
+```ruby
+class HomeController < ApplicationController
+  def index
+    http_cache_forever(public: params[:public], version: params[:version] || 'v1') do
```

Comment:
We need to explain the `public` parameter as well and also mention about the `last_modified` header which is set to `Time.new(2011, 1, 1).utc`. So this helper should be used with caution as browser/proxy won’t revalidate the resource back to the server unless force reload is done.